vendor/github.com/go-openapi/analysis/mixin.go

```diff
@@ -27,7 +27,7 @@ import (
 // collisions are avoided by appending "Mixin<N>" but only if
 // needed.
 //
-// The following parts of primary are never modified by merging:
+// The following parts of primary are subject to merge, filling empty details
```

Comment:
that's a terrifying godoc change for a point release. do we depend on the previous behavior?

Reply:
answering my own question, no calls to this in our codebase